cms/templates/videos_index.html

```diff
@@ -56,7 +59,7 @@ <h1 class="page-header">
             <h3 class="sr">${_("Page Actions")}</h3>
             <ul>
```

Comment:
Can we get rid of the ```<ul> and <li>``` markup here?   A list of one is not really a list at all.  It adds so much extra verbosity and keystrokes for screen reader users.